src/jit/rationalize.cpp

```diff
@@ -93,14 +93,33 @@ void Rationalizer::RewriteSIMDOperand(LIR::Use& use, bool keepBlk)
         return;
     }
 
-    // If we have GT_IND(GT_LCL_VAR_ADDR) and the GT_LCL_VAR_ADDR is TYP_BYREF/TYP_I_IMPL,
-    // and the var is a SIMD type, replace the expression by GT_LCL_VAR.
+    // If we have GT_IND(GT_LCL_VAR_ADDR) and the var is a SIMD type,
+    // replace the expression by GT_LCL_VAR or GT_LCL_FLD.
     GenTree* addr = tree->AsIndir()->Addr();
-    if (addr->OperIsLocalAddr() && comp->isAddrOfSIMDType(addr))
```

Comment:
That looks like before it could process both `GT_IND(GT_LCL_FLD_ADDR(GT_LCL_FLD))`  and `GT_IND(GT_LCL_VAR_ADDR(LCL_VAR))`, but now it is limited to the second option.
Was it intentional? Does it produce diffs?

Reply:
`GT_LCL_FLD_ADDR` was rejected by `isAddrOfSIMDType`. That's also the reason I got rid of the call to `isAddrOfSIMDType`.